pkgs/top-level/aliases.nix

```diff
@@ -527,6 +527,7 @@ mapAliases ({
   google-gflags = gflags; # Added 2019-07-25
   google-musicmanager = throw "google-musicmanager has been removed because Google Play Music was discontinued"; # Added 2021-03-07
   google-music-scripts = throw "google-music-scripts has been removed because Google Play Music was discontinued"; # Added 2021-03-07
+  google-play-music-desktop-player = throw "GPMDP shows a black screen, upstream homepage is dead, use 'ytmdesktop' instead"; # Added 2022-06-16
```

Comment:
You should also effectively remove the `all-packages.nix` call for this package, not only the `default.nix` expression.

Reply:
Missed the file during commit, thanks.